### ISSUE TITLE
grid Start failed

### ISSUE THREAD

**Issue by Dragon-Tab** [OpenMined/PySyft#6628]
When I execute  source .env && docker-compose up

ERROR: 
grid-backend-1 exited with code 1
grid-db-1              | ls: cannot access '/docker-entrypoint-initdb.d/': Operation not permitted
grid-db-1 exited with code 2

What should I do to solve this problem?

**Comment by madhavajay** [comment]
Hi @Dragon-Tab use `hagrid` to deploy rather than directly with docker compose. Alternatively if you run `hagrid` with `--cmd=true` it will output the command you can use to manually run with `docker compose`.

Also you will need `docker compose` v2 which is no longer called `docker-compose`. It ships with all the new docker desktop versions or you can get it for linux here: https://github.com/docker/compose/

TLDR:
```
$ pip install hagrid
$ hagrid launch domain to docker:8081 --tag=latest
```
